src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -155,6 +156,58 @@ public void addTag(Tag t) throws UniqueTagList.DuplicateTagException {
 
     //// util methods
 
+    /**
+     * Remove tag from Person if the tag exist in Person.
+     * @param tag
+     * @param person
+     */
+    private void removeTagFromPerson(Tag tag, Person person) {
```

Comment:
There is no update to the user guide for this feature. Remember, any new feature or change in functionality should ALWAYS be accompanied by an update to the tests and documentation.